adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -318,6 +288,12 @@ - (nullable NSError *)loadAd:(nonnull NSString *)placement
     return nil;
   }
 
+  // We already requested an ad for _prioritizedPlacementID,
+  // so we don't need to request again.
```

Comment:
Is thiis really ok? I see you pass this ID into init, but that doesn't necessarily mean an ad is cached. In fact, we already confirmed in the code above that the ad ISN'T cached. Shouldn't we still call loadPlacementWithID: and wait for a callback? What happens during a refresh or the next time an ad is requested for this placement?

Put another way, is there any reason we can't just fall through to the default behavior and call loadPlacementWithID: again? I'd prefer to get rid of the _prioritizedPlacementID in this class if we don't need it. I would assume Vungle SDK can handle an additional loadPlacementWithID: call.

Reply:
Hi Eric, I'm really sorry for the confusing, the `_prioritizedPlacementID` logic is a workaround of a bug for Vungle SDK sequential download with prioritized  placement:

If we add a  placementID in a dictionary then we pass this dictionary to the `options` parameter of new Vungle SDK init api: `- (BOOL)startWithAppId:options:error:`, Vungle SDK will initialize and begin to load this placementID. Then if we call `loadPlacementWithID:` method to load this placementID which is already beginning to load. The download process will stuck. 

So we add this `_prioritizedPlacementID` related logic, if a placementID is already beginning to load in the new Vungle SDK init api: `- (BOOL)startWithAppId:options:error:`, we will not load it again in the adapter to avoid stuck the download process. After the SDK initialization is finished,  the ` _prioritizedPlacementID = nil;`, so the refresh or the next time an ad is requested for this placement can work. 

We will remove this `_prioritizedPlacementID` logic in the next version after the stuck issue is fixed. Thanks so much. 😊

Reply:
I see. Can you update the comment to say something like:

// Vungle 6.7.0 SDK cannot handle a second `loadPlacementWithID:` call while the first ad load is in progress. Work around this behavior by explicitly avoiding calling `loadPlacementWithID:`on the first request for this placement, as the priority placement has already started loading at initialization time.

Reply:
@ericleich Hi Eric, got it, thanks a lot for the guidance, I will update the comment.  :)

Reply:
@ericleich Updated, thanks and have a great day. O(∩_∩)O